foodsaving/conversations/api.py

```diff
@@ -101,6 +106,35 @@ class ConversationMessageViewSet(
     def get_queryset(self):
         return self.queryset.filter(conversation__participants=self.request.user)
 
+    @detail_route(
+        methods=('POST',),
+        filter_fields=('name',)
+    )
+    def reactions(self, request, pk):
+        """route for POST /messages/{id}/reactions/ with body {\"name\":\"emoji_name\"}"""
+
+        instance = get_object_or_404(ConversationMessage.objects, id=pk)
+        reaction = ConversationMessageReaction(message=instance, name=request.data['name'], user=request.user)
+        try:
+            reaction.save()
+        except IntegrityError:
```

Comment:
I guess that happens when a user adds the same emoji two times?

Reply:
Indeed it does. It is changed in the last commit though.